### PR TITLE
#23993 repo layout, take 2

### PR DESCRIPTION
**Pull request by danchr**
This consolidates and reorganises a few modules:

- combine key & mapping into one, moved out of `service`
- move common out of `service`
- move some utility functions from `common` to `util`
- move cpr out of `service`

In addition, I noticed that my prior refactoring broke deployment from Jenkins, so this PR fixes that.

### CODE REVIEW

File: backend/mora/cpr.py [hunk 1]
```diff
@@ -15,10 +15,11 @@
...
 
-from mora import exceptions
-from mora import util
-from mora.integrations.serviceplatformen import get_citizen
-from mora.service import keys
+from . import exceptions
+from . import mapping
+from . import util
+from .integrations.serviceplatformen import get_citizen
+
```
[the remark this answers is not in the document]
Comment (danchr) [reply backend/mora/cpr.py]: It seems rather different than the others in that it doesn't do any actual LoRA-logic.

File: backend/.jenkins/common.sh [hunk 2]
```diff
@@ -3,6 +3,16 @@
 set -ex
 
 TOPDIR=$(cd "$(dirname ${BASH_SOURCE[0]})"/../..; pwd)
+if [[ -d "$TOPDIR/.git" ]]
+then
+    GIT_COMMIT=$(GIT_DIR="$TOPDIR/.git" git rev-parse --short HEAD)
+elif [[ -d "$TOPDIR/.hg" ]]
```
[the remark this answers is not in the document]
Comment (danchr) [reply backend/.jenkins/common.sh]: One of the changes to the Jenkinsfile — doing the checkout in a separate step — means that it no longer sets `$GIT_COMMIT`, so we calculate it ourselves.

[the remark this answers is not in the document]
Comment (cmoesgaard) [reply backend/.jenkins/common.sh]: I mean, why do we need support for Mercurial?

[the remark this answers is not in the document]
Comment (danchr) [reply backend/.jenkins/common.sh]: Quoting myself:

    jenkins scripts: support running from a hg-git repository

    I do this locally, so this allows me to try out the scripts without
    switching to another repository.

File: backend/mora/cpr.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (cmoesgaard) [reply backend/mora/cpr.py]: It's still a part of the services we're exposing, so I think it makes sense to keep it in the `service` directory. We have other service endpoints than also don't strictly do any LoRa-logic, e.g. the address auto completely endpoint, so an idea could be to keep all services of that kind in a separate subdirectory.

File: backend/.jenkins/common.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (cmoesgaard) [reply backend/.jenkins/common.sh]: If this code is specific to your setup, I don't think it belongs here.

[the remark this answers is not in the document]
Comment (danchr) [reply backend/.jenkins/common.sh]: Fair enough.

File: backend/mora/cpr.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (danchr) [reply backend/mora/cpr.py]: Ideally, I'd like to have that logic somewhere else, but I'll just revert this one for now.

**Pull request merged**